### ISSUE TITLE
Windows (dokany) support

### ISSUE THREAD

**Issue by vlad20012** [SerCeMan/jnr-fuse#10]
[Dokany](https://github.com/dokan-dev/dokany) (windows implementation of userspace filesystem) is currently support FUSE API. It means that jnr-fuse can use Docany as well as linux FUSE implementation. The problem is only in [this line](https://github.com/SerCeMan/jnr-fuse/blob/master/src/main/java/ru/serce/jnrfuse/AbstractFuseFS.java#L50) - library name. Docan library named as `dokanfuse1`. Think system property is good solution.

**Comment by SerCeMan** [comment]
> The problem is only in this line

Of course, it is not a problem to me to fix one line, but are you sure about it? I suppose, there are can be a lot of issues on windows. 
By the way, determining an OS at runtime is a better way to deal with it. Mac OS also has a different lib name.

**Comment by vlad20012** [comment]
> there are can be a lot of issues on windows.

Can be. But it is problems of Dokan.

**Comment by vlad20012** [comment]
I confess that I did not sufficiently studied the question. Some time ago I tried to use Dokan-java, at that time it was very bugly library and now it is not maintained at all. But now Dokany support FUSE API fully. And jnr-fuse looks workable. Think if FUSE API supported, any possible issues is issues of Docany FUSE implementations, not this library. And I do not need it now. Just I thought it might someone need.
